swig/cpp/src/Connection.h

```diff
@@ -31,10 +31,10 @@ extern "C" {
 #include "sysrepo.h"
 }
 
-class Connection:public Throw_Exception
+class Connection
 {
 public:
-    Connection(const char *app_name, const sr_conn_options_t opts = SESS_DEFAULT);
+    Connection(const char *app_name, const sr_conn_options_t opts = 0);
```

Comment:
It is because of the swig bindings, if I put SR_SESS_DEFAULT python bindings would not recognize it at runtime.

I had some difficulties generating wrapper code for the C structs and at the same time using them as default values in function overloading, so I created my custom data type based on sysrepo ones.

Since almost all of the default values use the first element ( value 0) I wanted to remove my custom enums which are used only for function overloading.

@jktjkt  you do make a valid point, I could use typedef or a define just for the default value that would make it more readable. At the moment I am not sure how swig will behave I will check this tomorrow.